src/utils/archive.ts

```diff
@@ -62,7 +63,7 @@ const Archive = {
   async edit ( doc: Document, data ) {
 
     const removeLines = _.uniqBy ( data.remove, line => line['lineNumber'] ) as any, //TSC
-          insertLines = _.sortBy ( _.map ( data.insert, ( text, lineNumber ) => ({ text, lineNumber }) ), [line => line.lineNumber] ).map ( line => line['text'] ), //TSC
+          insertLines = _.sortBy ( _.map ( data.insert, ( text, lineNumber ) => ({ text, lineNumber }) ), [line => extractDate(line.text)] ).map ( line => line['text']), //TSC
```

Comment:
I think we should sort todos by finished date, if they have one, then by line, basically moving those todos without a finishing date below those with one.

Reply:
Is there any type of task which gos to 'Archive' without finished data? I thought it is possible to move only 'done' or 'cancelled' tasks to 'archive'.